Fix "error: comma at end of enumerator list" in c89 mode

diff --git a/include/unwind.h b/include/unwind.h
index 58dc3d9..e7d088f 100644
--- a/include/unwind.h
+++ b/include/unwind.h
@@ -35,9 +35,11 @@ typedef enum {
   _URC_END_OF_STACK = 5,
   _URC_HANDLER_FOUND = 6,
   _URC_INSTALL_CONTEXT = 7,
-  _URC_CONTINUE_UNWIND = 8,
 #if defined(_LIBUNWIND_ARM_EHABI)
+  _URC_CONTINUE_UNWIND = 8,
   _URC_FAILURE = 9
+#else
+  _URC_CONTINUE_UNWIND = 8
 #endif
 } _Unwind_Reason_Code;
 
